routes/team/get-team.js

```diff
@@ -79,6 +79,13 @@ const handler = async (req, reply) => {
     return;
   }
 
+  // Check if user already applied for the team
+  let alreadyApplied = false;
+  if (team.applications && team.applications
+    .find(application => String(application.user._id) === String(authPayload._id))) {
```

Comment:
Sama homma kun äsken. Mä oon tehny näitä tyhmästi findilla, älä toista :D

Reply:
Done